xbmc/storage/MediaManager.cpp

```diff
@@ -64,23 +64,21 @@ const char MEDIA_SOURCES_XML[] = { "special://profile/mediasources.xml" };
 CMediaManager::CMediaManager()
 {
   m_bhasoptical = false;
-  m_platformStorage = nullptr;
 }
 
 void CMediaManager::Stop()
 {
   if (m_platformStorage)
     m_platformStorage->Stop();
 
-  delete m_platformStorage;
-  m_platformStorage = nullptr;
+  m_platformStorage.reset();
```

Comment:
Is this actually needed? Or is it safe to free upon dtor?

Reply:
It should be safe on destruction but since the members (and workflow from CApp) are controlled in `Initialize` and `Stop` methods I prefer to keep it as is. Its not my goal to change anything in storage but disc handling only.

Reply:
done in the last commit...hopefully right.
Let's wait for jenkins to be red :)